extensions/aws/processors/PutS3Object.h

```diff
@@ -30,6 +30,7 @@
 #include <utility>
 #include <vector>
 
+#include "io/StreamPipe.h"
```

Comment:
No stream pipes in this file either.

Reply:
InputStreamCallback is defined in that header.

Reply:
Oh, true